build-system/tasks/process-github-issues.js

```diff
@@ -115,220 +113,199 @@ function getIssues(opt_page) {
-                issueType === 'Type: Status Update'
-              ) {
-                issueNewMilestone = MILESTONE_DOCS_UPDATES;
-                updates.push(applyMilestone(issue, issueNewMilestone));
-              } else if (
-                issueType === 'Type: Bug' ||
-                issueType === 'Related to: Flaky Tests'
-              ) {
-                issueNewMilestone = MILESTONE_BACKLOG_BUGS;
-                updates.push(applyMilestone(issue, issueNewMilestone));
-              } else if (milestone == null) {
-                updates.push(applyMilestone(issue, issueNewMilestone));
-              }
-            }
-          } else if (milestone == null) {
-            updates.push(applyMilestone(issue, issueNewMilestone));
-          } else if (
-            issueNewMilestone === MILESTONE_PRIORITIZED_FRS ||
-            issueNewMilestone === MILESTONE_NEW_FRS
-          ) {
-            updates.push(applyLabel(issue, 'Type: Feature Request'));
-          } else if (
-            issueNewMilestone === MILESTONE_BACKLOG_BUGS ||
-            milestoneTitle.startsWith('Sprint')
-          ) {
-            updates.push(applyLabel(issue, 'Type: Bug'));
-          }
-          // Apply default priority if no priority
-          if (
-            hasPriority == false &&
-            issueNewMilestone != MILESTONE_NEW_FRS &&
-            issueNewMilestone !== MILESTONE_3P_IMPLEMENTATION &&
-            issueNewMilestone !== MILESTONE_PENDING_TRIAGE &&
-            milestone != null
-          ) {
-            updates.push(applyLabel(issue, 'P2: Soon'));
-          }
-          // Add comment with missing Category
-          if (hasCategory == false) {
-            if (
-              issueNewMilestone === MILESTONE_PENDING_TRIAGE ||
-              issueNewMilestone === MILESTONE_DOCS_UPDATES ||
-              issueNewMilestone == null ||
-              issueNewMilestone === MILESTONE_GREAT_ISSUES
-            ) {
-              if (isDryrun) {
-                log(colors.green('No comment needed  for #' + issue.number));
-              }
-            } else {
-              updates.push(
-                applyComment(
-                  issue,
-                  "This issue doesn't have a category" +
-                    ' which makes it harder for us to keep track of it. ' +
-                    assigneeName +
-                    ' Please add an appropriate category.'
-                )
-              );
-            }
-          }
-          return Promise.all(updates);
-        });
-      });
-      return promise;
-    });
+      } else {
+        updates.push(
+          applyComment(
+            issue,
+            "This issue doesn't have a category" +
+              ' which makes it harder for us to keep track of it. ' +
+              assigneeName +
+              ' Please add an appropriate category.'
+          )
+        );
+      }
+    }
+
+    await Promise.all(updates);
```

Comment:
I suspect we should just `await` each update in-place, but not going to address that in this PR

Reply:
Side note: It has been a couple of years since we used `build-system/tasks/process-github-issues.js` (it used to be run manually in 2018), so we could ignore this file and delete it in a separate PR.

Reply:
Looking at it, I got that sense. I'll revert this one and create a separate PR to remove.